meta-mender-demo/recipes-mender/example-state-scripts/files/example-script

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 
-echo "$(cat /etc/mender/artifact_info): $(basename "$0") was called!" >&2
+echo "$(mender -show-artifact): $(basename "$0") was called!" >&2
```

Comment:
Thanks, good catch!

However, there is a subtle difference in behavior in `-show-artifact`, namely that it will not be updated until the artifact is committed. The old `artifact_info` file would be updated as soon as the device was rebooted. Some of the point of the example script is to show this difference, therefore I suggest that you add one more line to the recipe file, so that we also see the new name after it has been committed:

```
cp example-script ${MENDER_STATE_SCRIPTS_DIR}/ArtifactCommit_Leave_50
```

Reply:
Ah indeed, thanks for this feedback, I will do so.

Reply:
Updated ;)